source/state_representation/test/tests/space/cartesian/test_cartesian_state.cpp

```diff
@@ -1083,3 +1084,20 @@ TEST(CartesianStateTest, TestSubtractionOperators) {
   //wrench -= twist;
   //wrench -= acc;
 }
+
+TEST(CartesianStateTest, TestUtilities) {
+  auto state_variable_type = string_to_cartesian_state_variable("position");
+  EXPECT_EQ(state_variable_type, CartesianStateVariable::POSITION);
+  EXPECT_EQ("position", cartesian_state_variable_to_string(CartesianStateVariable::POSITION));
+  EXPECT_THROW(string_to_cartesian_state_variable("foo"), exceptions::InvalidStateVariableException);
+
+  auto state = CartesianState();
+  state.set_position(std::vector<double>{1.0, 2.0, 3.0});
+  EXPECT_TRUE(state.get_position().cwiseEqual(state.get_state_variable(CartesianStateVariable::POSITION)).all());
```

Comment:
It's a detail but I like the python tests better, we should compare numerical values to the `get_state_variable` and not the `get_position` because we know internally that it's the same

Reply:
yup, fair enough